### PR TITLE
update UI

### PR DESCRIPTION
**Pull request by Quramy**
## What I did

- update UI (https://www.figma.com/file/2SqjEDoQyhm0QaRYfv0tpR/reg-viz?node-id=902%3A1)

See also #9

### CODE REVIEW

File: packages/frontend/public/static.css [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+  --color-border: var(--color-gray200);
+
+  /* gutter */
+  --gutter-base: 8px;
+
+  /* focusing */
+  --shadow-focus: rgba(255,68,56,0.4) 0px 0px 0px 4px;
+
+  /* animation */
+  --transition-base: all .15s ease-in-out;
```
Comment (tsuyoshiwada) [inline packages/frontend/public/static.css]:
```suggestion
  --transition-base: all .15s cubic-bezier(0.4, 0.0, 0.2, 1);
```

:pray:

File: packages/frontend/public/static.css [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
   color: inherit;
   text-decoration: none;
 }
 
 a.text-link {
-  color: #4C89C3;
+  color: #2582C7;
+  text-decoration: underline;
   transition-duration: color;
 }
```
Comment (tsuyoshiwada) [inline packages/frontend/public/static.css]:
```suggestion
a:hover {
  text-decoration: none;
}
```

File: packages/frontend/src/components/client-id-modal/client-id-modal.css [hunk 3]
```diff
@@ -1,17 +1,121 @@
...
+  margin: 24px 0;
   word-break: break-all;
-  white-space: pre-wrap;
+  white-space: pre;
   border-radius: 3px;
-  background-color: #191919;
-  padding: 15px 30px;
+  background-color: var(--color-secondary);
+  padding: 24px;
   font-family: SFMono-Regular, Consolas, "Liberation Mono", Menlo, Courier, monospace;
```
Comment (tsuyoshiwada) [inline packages/frontend/src/components/client-id-modal/client-id-modal.css]: Please use Roboto Mono Regular :pray:
see https://www.figma.com/file/2SqjEDoQyhm0QaRYfv0tpR/reg-viz?node-id=872%3A3&viewport=362%2C111%2C0.499459445476532

File: packages/frontend/src/components/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div className={styles.header}>
+      <nav>
+        <a className={styles.brandLogo} href={LINKS.organization} />
+      </nav>
+      <a href={LINKS.configureInstallations}>Configure</a>
+      <a href="#" onClick={handleOnLogout}>Logout</a>
+      <a href={LINKS.ghAppRepository}>
+        <Icon name="github" size="big" />
+      </a>
```
Comment (tsuyoshiwada) [inline packages/frontend/src/components/header/header.tsx]: Point

1. Please set `:hover` styles, using `opacity: 0.7`.
2. Please set `:hover` styles, using `text-decoration: underline; color: var(--color-priary)`.

File: packages/frontend/src/components/client-id-modal/client-id-modal.css [hunk 5]
```diff
@@ -1,17 +1,121 @@
...
+}
+
+.closeButton {
+  cursor: pointer;
+  position: absolute;
+  top: -18px;
+  right: -20px;
+  background-color: transparent;
+  border: none;
+}
```
Comment (tsuyoshiwada) [inline packages/frontend/src/components/client-id-modal/client-id-modal.css]:
```css
.closeButton:hover {
  opacity: 0.8;
}
```

:bow:

**Comment by Quramy** [comment]
@tsuyoshiwada Thanks for your review and building design prototype ! 

I fixed your points, pls check them 🙏

**Pull request merged**